daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -338,6 +344,44 @@ object Node {
 
   }
 
+  final case class NodeRollback[+Nid, +Cid](
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      // Figure-out what information needs to be contained in a Rollback node.
+      // For the moment, we just have the children.
+      children: ImmArray[Nid]
+  ) extends GenNode[Nid, Cid]
+      with NodeInfo.Rollback {
+
+    def templateId: TypeConName =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
+
+    def version: TransactionVersion =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
+
+    private[lf] def updateVersion(version: TransactionVersion): NodeLookupByKey[Cid] =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
+
+    def byKey: Boolean =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
+
+    def versionedChosenValue: Value.VersionedValue[Cid] =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
+
+    def versionedExerciseResult: Option[Value.VersionedValue[Cid]] =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
+
+    def versionedKey: Option[KeyWithMaintainers[Value.VersionedValue[Cid]]] =
+      // TODO https://github.com/digital-asset/daml/issues/8020
+      sys.error("rollback nodes are not supported")
```

Comment:
I think those keys methods override nothing. 
```suggestion
```

Reply:
We need `byKey` to fulfil the `GenNode` interface, but I removed the other 3.